regparse.c

```diff
@@ -37,6 +37,7 @@
 
 #include "regparse.h"
 #include <stdarg.h>
+#include "internal/sanitizers.h"
```

Comment:
Because of this addition we need to run `ruby tool/update-deps --fix` (this is the cause of CI failing). It's a little finicky, ping me if you'd prefer I run it to fix the deps 😅.